kitsune/flagit/views.py

```diff
@@ -163,9 +171,26 @@ def build_hierarchy(parent_id=None, level=0):
 
 
 @group_required("Content Moderators")
+@require_http_methods(["GET", "POST"])
 def moderate_content(request):
     """Display flagged content that needs moderation."""
     product_slug = request.GET.get("product")
+    assignee = request.GET.get("assignee")
+
+    if (
+        assignee
+        and not User.objects.filter(
+            is_active=True, username=assignee, groups__name="Content Moderators"
+        ).exists()
+    ):
+        return HttpResponseNotFound()
+
+    if request.method == "POST":
```

Comment:
This can be merged with the other check whether the request.method equals to POST.

Reply:
Yeah, that's cleaner. I was trying to avoid the bit of work involved with getting the `content_type` and setting up the query for `objects` if the POST was invalid.